examples/acquiring_data/search_cdaweb.py

```diff
@@ -0,0 +1,44 @@
+"""
+========================
+Getting data from CDAWeb
+========================
+
+How to download data from the Coordinated Data Analysis Web (CDAWeb).
+
+CDAWeb stores data from from current and past space physics missions, and is
+full of heliospheric insitu datasets.
+"""
+
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+from sunpy.net.cdaweb.attrs import Dataset
```

Comment:
You shouldn't need to do this, you can have the client register the attrs namespace so this becomes `a.cdaweb.Dataset`, 
like this:

https://github.com/sunpy/sunpy/blob/71b09358db2a08fa2aec34a9798c1cc7ed6368ac/sunpy/net/jsoc/jsoc.py#L806-L808

though why this isn't in the Fido client dev docs is a mystery to me.

Reply:
Thanks - FWIW I copy and pasted across code I wrote ~1 year ago and adapted it for sunpy, so haven't been following up to date Fido client docs. I'll take a look through the docs though and see if there's anything else I should update/change.